hera_cal/utils.py

```diff
@@ -0,0 +1,33 @@
+import os
+def get_HERA_aa(freqs,calfile='hera_cm',
+                locations_file=None,
+                array_epoch_jd=None):
+    # create an aipy AntennaArray object with the HERA configuration as it
+    # was on array_epoch_jd.
+    #
+    # A default file is supplied but will soon be out of date.
+    #   - create a new one with hera_mc/scripts/XXX.py
+    #
+    # NOTE, if you didn't specify a array_epoch_jd, one will be chosen for you
+    # by the cal file.
```

Comment:
Yeah, having defaults in two places is confusing. Changed so that all defaults are in the cal file, this function is now just limited to handing the case of defaulting to a base cal file which uses extra options.  

Hopefully the docstring is now the most helpful get_aa docstring ever written.